src/util.js

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
Change request: We already have `src/utils.js` so I wouldn't add a new one.

All other query param serialization code is in `src/params_serialization` so maybe these function should be moved there?

Reply:
The idea behind the new file was to have the public utils there. Then again, the interface exposing file can decide that. I'll see which file suits better and switch to that.